aerogear-windows-push/UPSHttpClient.cs

```diff
@@ -45,13 +45,28 @@ public UPSHttpClient(Uri uri, string username, string password)
 
         public async Task<HttpStatusCode> register(Installation installation)
         {
+            return await Request(installation, delegate(HttpWebRequest request) { });
+        }
+
+        public async Task<HttpStatusCode> register(Installation installation, string pushMetricsId)
+        {
+
+            return await Request(installation, delegate(HttpWebRequest request)
+            {
+                request.Headers["push-identifier"] = pushMetricsId;
```

Comment:
See https://github.com/aerogear/aerogear-unifiedpush-server/pull/543 - we now have a different API for that.

Metrics is done by PUT agains a separate endpoint